scripts/run-travis-tests.sh

```diff
@@ -17,7 +17,8 @@ cd build
 time cmake .. \
       -DCHAINER_COMPILER_ENABLE_PYTHON=ON \
       -DPYTHON_EXECUTABLE=/usr/bin/python3 \
-      -DCHAINER_COMPILER_ENABLE_OPENCV=ON
+      -DCHAINER_COMPILER_ENABLE_OPENCV=ON \
+      -DCHAINER_COMPILER_PREBUILT_CHAINERX_DIR=/usr/local/lib/python3.6/dist-packages/chainerx
```

Comment:
```suggestion
      -DCHAINER_COMPILER_PREBUILT_CHAINERX_DIR=$(pip3 show chainer | grep Location | sed -e 's/Location: //')/chainerx
```

Seems more general(sorry for dirty shell stuff).

Reply:
It worked nicely (I slightly golfed the shell script though), thanks!